test/merge-patient-lab-iips-mocha.js

```diff
@@ -5,151 +5,273 @@ var chaiAsPromised = require('chai-as-promised');
 chai.should();
 chai.use(chaiAsPromised);
 
+var sinon = require('sinon');
+
 var _ = require('underscore');
 var noflo = require('noflo');
+
+var componentFactory = require('../components/merge-patient-lab-iips');
 var getComponent = require('../components/merge-patient-lab-iips').getComponent;
+var State = require('../components/state');
```

Comment:
We can't have it both ways.  David is telling me that the state is the input to the inputState API.  You're telling me to just pas the values in there (as I originally did).  This may need a discussion.